client/src/components/User/DiskUsage/Management/Cleanup/ReviewCleanupDialog.vue

```diff
@@ -0,0 +1,252 @@
+                <b-form-checkbox
+                    v-model="selectedItems"
+                    :checked="allSelected"
+                    :key="data.index"
+                    :value="data.item"></b-form-checkbox>
+            </template>
+            <template v-slot:cell(update_time)="data">
+                <UtcDate :date="data.value" mode="elapsed" />
+            </template>
+        </b-table>
+        <template v-slot:modal-footer>
+            <b-pagination v-if="hasPages" v-model="currentPage" :total-rows="totalRows" :per-page="perPage" />
+            <b-button
+                :disabled="!hasItemsSelected"
+                :variant="deleteButtonVariant"
+                v-b-modal.confirmation-modal
+                class="mx-2"
+                data-test-id="delete-button">
+                {{ permanentlyDeleteText }} {{ deleteItemsText }}
+            </b-button>
+        </template>
+
+        <b-modal
+            id="confirmation-modal"
+            :title="confirmationTitle"
+            title-tag="h2"
+            :ok-title="permanentlyDeleteText"
+            :ok-variant="confirmButtonVariant"
+            :ok-disabled="!confirmChecked"
+            @show="resetConfirmationModal"
+            @ok="onConfirmCleanupSelectedItems"
+            static
+            centered>
+            <b-form-checkbox id="confirm-delete-checkbox" v-model="confirmChecked" data-test-id="agreement-checkbox">
+                {{ agreementText }}
+            </b-form-checkbox>
+        </b-modal>
+    </b-modal>
+</template>
+
+<script>
+import _l from "utils/localization";
+import { bytesToString } from "utils/utils";
+import UtcDate from "components/UtcDate";
+import { CleanupOperation } from "./model";
+
+export default {
+    components: {
+        UtcDate,
+    },
+    props: {
+        operation: {
+            type: CleanupOperation,
+            required: false,
+            default: null,
+        },
+        totalItems: {
+            type: Number,
+            required: false,
+            default: 0,
+        },
+        show: {
+            type: Boolean,
+            required: false,
+        },
+    },
+    created() {
+        this.showDialog = this.show;
+    },
+    data() {
+        return {
+            fields: [
+                {
+                    key: "selected",
+                    label: "",
+                    sortable: false,
+                },
+                {
+                    key: "name",
+                    sortable: true,
+                },
+                {
+                    key: "size",
+                    sortable: false, // The API does not allow sort by size?
+                    formatter: this.toNiceSize,
+                },
+                {
+                    label: "Updated",
+                    key: "update_time",
+                    sortable: true,
+                },
+            ],
+            sortBy: "update_time",
+            sortDesc: true,
+            perPage: 5,
```

Comment:
That is too small to be usable IMO
```suggestion
            perPage: 50,
```

Reply:
Ups... this is a leftover from testing the pagination, thanks for the catch!